core/src/main/java/io/questdb/griffin/engine/groupby/vect/AvgDoubleVectorAggregateFunction.java

```diff
@@ -60,9 +60,9 @@ public AvgDoubleVectorAggregateFunction(int keyKind, int columnIndex, int worker
     }
 
     @Override
-    public void aggregate(long address, long addressSize, int columnSizeHint, int workerId) {
+    public void aggregate(long address, long frameRowCount, int workerId) {
```

Comment:
Nice change here! That frame row count calculation based on the size in bytes was silly.